data/helpers.d/network

```diff
@@ -0,0 +1,9 @@
+# Check availability of a web path
+#
+# example: ynh_path_validity $domain$path
+#
+# usage: ynh_path_validity $domain_and_path
+# | arg: domain_and_path - complete path to check
+ynh_path_validity () {
```

Comment:
To me it looks like if `app` is defined, then some settings of the app are changed/set : https://github.com/YunoHost/yunohost/blob/unstable/src/yunohost/app.py#L947-L949

So there are side-effects, but only if `app` is defined. And as was trying to point out in the previous comment, your helper implicitly require `$app` (in bash this time) is defined. So bottom line is : if you remove the `-a $app`, then everything should be fine 😄 !

As a side note though, it's not really cool that the python throws an exception if the path is not available. Not sure if this is considered as a side effect... But that's basically the same kind of situation which led to the dirty `no_log` helper :/.. We should really refactor these functions :/